src/parser/csql_grammar.y

```diff
@@ -15769,7 +15769,13 @@ limit_factor
                         PARSER_SAVE_ERR_CONTEXT ($$, @$.buffer_pos)
 
                 DBG_PRINT}}
+        | identifier    /* for PL/CSQL Static SQL only */
```

Comment:
Is '.' allowed in variable names in PL/CSQL?
Otherwise, use identifier_without_dot

Also, you should consider that identifiers such as `aaa` and [aaa] are also allowed.

Reply:
> Otherwise, use identifier_without_dot
Good point. Thank you.

> Also, you should consider that identifiers such as `aaa` and [aaa] are also allowed.
Identifiers can be one of the forms "...", [...], `...` as in CUBRID SQL.